docker-compose-tests.yml

```diff
@@ -8,6 +8,7 @@ services:
       args:
         NODE_ENV: test
     command: sh -c 'npm run git-hook && npm run snyk-test && npm run watch-test'
+    user: root
```

Comment:
This shouldn't be needed if `prepublish` is removed from `package.json`. But the line above i.e. command: sh -c 'npm run git-hook && npm run snyk-test && npm run watch-test' will also need to be updated to remove `&& npm run snyk-test ` which is probably a good thing. I'm not sure I see the value in running `snyk-test` just the once at the start of the development process. It runs as part of `prepush` so any issues will be caught there.